src/objects/page/page.twig

```diff
@@ -1,8 +1,13 @@
 <div class="o-page{% if class %} {{ class }}{% endif %}">
-  <div class="o-page__footer">
-    {% block footer %}{% endblock %}
-  </div>
-  <div class="o-page__content">
+  {% if block("header") %}
```

Comment:
According to [the Twig docs](https://twig.symfony.com/doc/3.x/functions/block.html), this should be a little more explicit...

```suggestion
  {% if block("header") is defined %}
```

If you already tried that and it didn't work, disregard.

Reply:
I merged this suggestion, but ended up removing it — with `is defined`, the header div was always rendered, regardless of whether it has content. When I leave off `is defined`, it behaves as expected — only rendering the header div if there's content in the header block.

Reply:
No worries, thanks for trying!